serve.go

```diff
@@ -43,17 +43,17 @@ func serve(sessionID string) {
 			conn, err := net.Dial("unix", path.Join(dir, clientExitSock))
 			if err == nil {
 				conn.Write([]byte(diagnostics))
-			} else {
-				timestamp := time.Now().UTC().Format(time.RFC3339)
-				diagnostics = fmt.Sprintf("At %s...\n\n%s", timestamp, diagnostics)
-				ioutil.WriteFile(path.Join(dir, "crash"), []byte(diagnostics), 0666)
 			}
+
+			timestamp := time.Now().UTC().Format(time.RFC3339)
+			diagnostics = fmt.Sprintf("At %s...\n\n%s", timestamp, diagnostics)
+			ioutil.WriteFile(path.Join(dir, "crash"), []byte(diagnostics), 0666)
```

Comment:
Woah, I just figured this out too. Coincidence

Reply:
Ahaha. I had it before, but I think it was deleted by a manually-assisted rebase :P